### ISSUE TITLE
`realpath': No such file or directory @ realpath_rec projectpath/    1|       |import Foundation (Errno::ENOENT)

### ISSUE THREAD

**Issue by swati-daffodilsw** [SlatherOrg/slather#393]
Hi,
Can anyone help me to resolve this issue?

**Comment by dunyakirkali** [comment]
I'm having the same issue on 9.3 :(

**Comment by swati-daffodilsw** [comment]
Also , When I create a Cocoa touch framework project in ios and integrating slather, I am getting following error:

error: **/appname.framework/appname: Failed to load coverage: No coverage data found
Traceback (most recent call last):
	17: from /usr/local/bin/slather:23:in `<main>'
	16: from /usr/local/bin/slather:23:in `load'
	15: from /usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/bin/slather:17:in `<top (required)>'
	14: from /usr/local/lib/ruby/gems/2.5.0/gems/clamp-0.6.5/lib/clamp/command.rb:132:in `run'
	13: from /usr/local/lib/ruby/gems/2.5.0/gems/clamp-0.6.5/lib/clamp/command.rb:67:in `run'
	12: from /usr/local/lib/ruby/gems/2.5.0/gems/clamp-0.6.5/lib/clamp/subcommand/execution.rb:11:in `execute'
	11: from /usr/local/lib/ruby/gems/2.5.0/gems/clamp-0.6.5/lib/clamp/command.rb:67:in `run'
	10: from /usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/lib/slather/command/coverage_command.rb:59:in `execute'
	 9: from /usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/lib/slather/command/coverage_command.rb:97:in `post'
	 8: from /usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/lib/slather/coverage_service/simple_output.rb:17:in `post'
	 7: from /usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/lib/slather/project.rb:98:in `coverage_files'
	 6: from /usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/lib/slather/project.rb:125:in `profdata_coverage_files'
	 5: from /usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/lib/slather/project.rb:125:in `each'
	 4: from /usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/lib/slather/project.rb:128:in `block in profdata_coverage_files'
	 3: from /usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/lib/slather/project.rb:128:in `map'
	 2: from /usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/lib/slather/project.rb:132:in `block (2 levels) in profdata_coverage_files'
	 1: from /usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/lib/slather/profdata_coverage_file.rb:197:in `ignored?'
/usr/local/lib/ruby/gems/2.5.0/gems/slather-2.4.5/lib/slather/profdata_coverage_file.rb:45:in `source_file_pathname': undefined method `each' for nil:NilClass (NoMethodError)

**Comment by dunyakirkali** [comment]
@swati-daffodilsw you should probably be creating a separate issue for that ;)

**Comment by swati-daffodilsw** [comment]
ok.Thanks for your suggestion.  BTW, Did you find any workaround for the above issue?

**Comment by dunyakirkali** [comment]
not yet :(

**Comment by germangb** [comment]
The error may be related to issue #376, which was fixed in #377

**Comment by mkchoi212** [comment]
I'm still experiencing the issue on Xcode 9.3.1., slather 2.4.5 😢 

`Errno::ENOENT: No such file or directory @ realpath_rec - /Users/mikechoi/project_name/    1|       |`

**Comment by dunyakirkali** [comment]
@germangb I still have the issue, even with the fix from #377

**Comment by swati-daffodilsw** [comment]
@germangb : I am also still facing the issue after #376,

**Comment by germangb** [comment]
@dunyakirkali @swati-daffodilsw Have you tried #398? Seems to fix this issue but hasn't been merged yet

**Comment by dunyakirkali** [comment]
@germangb nope not yet. Will check when I have the time

**Comment by dunyakirkali** [comment]
@germangb I can confirm that it works now.

**Comment by ksuther** [comment]
#398 has been merged.

**Comment by dunyakirkali** [comment]
@ksuther When could we expect a release?

**Comment by ksuther** [comment]
Soon, but it would be nice to get some additional confirmation from others that everything is working.

**Comment by dunyakirkali** [comment]
@swati-daffodilsw Could you confirm wether this fix works for u or not?

**Comment by swati-daffodilsw** [comment]
I haven't checked it yet. I will confirm you soon ok.

Best Regards,

Swati Agarwal | Senior Associate - IT

Contact: +01246630131 | Skype: swati.daffodilsw

Daffodil Software Pvt Ltd.15th Floor, Building A3

DLF Silokhera SEZ, Sector-30, Gurgaon -122001

www.daffodilsw.com


IMPORTANT: Information security policy of ​Daffodil Software Private Limited
  (in short DSPL) prohibits any unauthorized use and transfer of e-mail and
internet for transmission of information and data. If you are not the
intended person of this email communication and its attachments or you have
got it from another source, this is unauthorized and  DSPL does not claim
its ownership and correctness.  If you are not the named addressee you
should not forward, disseminate, distribute or copy this e-mail in any
form. Please notify the sender immediately by e-mail if you have received
this e-mail by mistake and delete this e-mail from your system.

=============================================================================================================================
